BREAKING.md

```diff
@@ -29,6 +31,14 @@ The `@fluid-experimental/tinylicious-client` package has been renamed to now be
 ### `@fluid-experimental/fluid-static` package name changed
 The `@fluid-experimental/fluid-static` package has been renamed to now be `@fluidframework/fluid-static`.
 
+### `FlushMode` enum values renamed
+`FlushMode` enum values from `@fluidframework/runtime-definitions` have ben renamed as following:
+- `FlushMode.Manual` to `FlushMode.TurnBased`
+- `FlushMode.Automatic` to `FlushMode.Immediate`
+
+### `FlushMode` default changed for `ContainerRuntime`
+Previously, `ContainerRuntime` had the default flush mode as `FlushMode.Immediate` (formerly `FlushMode.Automatic`). This default has been changed to `FlushMode.TurnBased` (formerly `FlushMode.Manual`). Note that the `batchBegin` and `batchEnd` events from `IContainerRuntime` will no longer be emitted for each processed op by default.
```

Comment:
It will be hard for people to consume that paragraph, thus I'd separate (across couple versions) rename vs. change in behavior.

`batchBegin` and `batchEnd` - This reads scary, as summarizer depends on it, to be able to inject summary at random time. So the first time I read it, I felt like everything will break :).
I think we want to soften message and say that natural batch granularity is extended because of these changes, i.e. batch scope used to be (on average) 1 op vs. now it's likely to be larger (scenario dependent).

Reply:
Thanks for the feedback, will redo. Can you look at https://github.com/microsoft/FluidFramework/pull/7071 where these changes are staged for merging? They are brought here in this PR only so that I can move forward with the batchmanager work.